test/scripts/tags/asset_img.js

```diff
@@ -41,7 +41,7 @@ describe('asset_img', () => {
   });
 
   it('default', () => {
-    assetImg('bar title').should.eql('<img src="/foo/bar" class="" title="title">');
+    assetImg('bar "a title"').should.eql('<img src="/foo/bar" class="" title="a title">');
```

Comment:
@SukkaW 

https://github.com/hexojs/hexo-util/commit/622167f3cf98f08436b0fb0bc1a7bc7ec00904f4 could have undesired output when

``` js
htmlTag('a', { src: '/foo/', title: 'title', alt: 'alt' }, 'text')
```

`title` and `alt` are incorrectly seen as boolean. It's a _very rare_ edge case (`title="title"` is valid but bad practice), so I don't mind keeping https://github.com/hexojs/hexo-util/commit/622167f3cf98f08436b0fb0bc1a7bc7ec00904f4, just fyi.